### PR TITLE
Py updates, fix env variable for timeout, add option to skip remote updates in build_updates script

### PR DESCRIPTION
**Pull request by satmandu**
- also rebuilds `parted` and updates `nconvert`.
##
Builds:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l`
##
Tested & Working properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 
##
### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/satmandu/chromebrew.git CREW_BRANCH=py_updates crew update \
&& yes | crew upgrade
```

### CODE REVIEW

File: tools/build_updated_packages.rb [hunk 1]
```diff
@@ -15,7 +15,9 @@
 
 abort "\nGITLAB_TOKEN environment variable not set.\n".lightred if ENV['GITLAB_TOKEN'].nil?
 abort "\nGITLAB_TOKEN_USERNAME environment variable not set.\n".lightred if ENV['GITLAB_TOKEN_USERNAME'].nil?
+puts "Setting the CREW_AGREE_TIMEOUT_SECONDS environment variable to less than the default of #{CREW_AGREE_TIMEOUT_SECONDS} may speed this up...".orange if ENV['CREW_AGREE_TIMEOUT_SECONDS'].nil?
 
+SKIP_UPDATE_CHECKS = ARGV.include?('--skip')
```
Comment (uberhacker) [inline tools/build_updated_packages.rb:20]: Should this be documented somewhere?

File: tools/build_updated_packages.rb [hunk 2]
```diff
@@ -161,13 +167,12 @@ def self.agree_default_yes(message = nil)
     puts "#{name.capitalize} #{@version} has no binaries and may not need them.".lightgreen
     next pkg
   else
-    build = compatibility == 'all' ? %w[x86_64 armv7l i686] : compatibility.split
-    build.each do |arch|
+    check_builds = compatibility == 'all' ? %w[x86_64 armv7l i686] : compatibility.split
```
Comment (uberhacker) [inline tools/build_updated_packages.rb:170]: This assumes all the compatibility lines are separated by spaces.
```
cage.rb:  compatibility 'aarch64, armv7l, x86_64'
```

File: tools/build_updated_packages.rb (hunk 1 above)
Comment (satmandu) [reply tools/build_updated_packages.rb:23]: Done.

Comment (uberhacker) [reply tools/build_updated_packages.rb:23]: I mean like in help.   It sorta already is self-documented here.

Comment (satmandu) [reply tools/build_updated_packages.rb:23]: It's only being used in that one script. It's not an addition to `crew` or to `const.rb`.

File: tools/build_updated_packages.rb (hunk 2 above)
Comment (satmandu) [reply tools/build_updated_packages.rb]: Good call.

Now it handles that case.

File: tools/build_updated_packages.rb (hunk 1 above)
Comment (uberhacker) [reply tools/build_updated_packages.rb:23]: Would it make sense to add to crew eventually?

File: tools/build_updated_packages.rb [hunk 3]
```diff
@@ -161,13 +170,12 @@ def self.agree_default_yes(message = nil)
     puts "#{name.capitalize} #{@version} has no binaries and may not need them.".lightgreen
     next pkg
   else
-    build = compatibility == 'all' ? %w[x86_64 armv7l i686] : compatibility.split
-    build.each do |arch|
+    check_builds = compatibility == 'all' ? %w[x86_64 armv7l i686] : compatibility.delete(',').split
```
Comment (uberhacker) [inline tools/build_updated_packages.rb:173]: That probably doesn't work for this?
```
snort.rb:  compatibility 'aarch64,armv7l,x86_64'
```

File: tools/build_updated_packages.rb (hunk 1 above)
Comment (satmandu) [reply tools/build_updated_packages.rb:23]: Maybe? If we eventually move the script into crew...

But I don't think it is needed at the moment.

File: tools/build_updated_packages.rb (hunk 3 above)
Comment (satmandu) [reply tools/build_updated_packages.rb:173]: Was easier just to change the packages...

**Review by uberhacker — APPROVED** [review]

**Review by uberhacker — APPROVED** [review]

**Pull request merged**